config/binoculars/config.yaml

```diff
@@ -0,0 +1,9 @@
+grpcPort: 50052
+httpPort: 8082
+metricsPort: 9002
+corsAllowedOrigins: ["http://localhost:3000", "http://localhost:8089"]
```

Comment:
Is there any concerns about setting this by default? We don't set corsAllowedOrigins default values in other configs.